itou/templates/approvals/printable_approval.html

```diff
@@ -29,35 +29,36 @@
                     <span class="d-block font-weight-bold">Votre structure :</span>
                     <span class="d-block">{{ siae.display_name|title }}</span>
                     {% if siae.address_on_one_line %}
-                        <span class="d-block">{{ siae.address_line_1 }}</span>
-                        <span class="d-block">{{ siae.address_line_2 }}</span>
-                        <span class="d-block">{{ siae.post_code }} - {{ siae.city }}</span>
+                        <address class="d-block">
+                            {{ siae.address_line_1 }}
+                            <br>
+                            {% if approval.user.address_line_2 %}
```

Comment:
Ça devrais pas plutôt être ça ?
```suggestion
                            {% if siae.address_line_2 %}
```